samples/csharp_dotnetcore/13.core-bot/Startup.cs

```diff
@@ -20,9 +20,6 @@ public void ConfigureServices(IServiceCollection services)
         {
             services.AddMvc().SetCompatibilityVersion(CompatibilityVersion.Version_2_1);
 
-            // Create the credential provider to be used with the Bot Framework Adapter.
-            services.AddSingleton<ICredentialProvider, ConfigurationCredentialProvider>();
```

Comment:
Based on this change, I think the ConfigurationCredentialProvider class in the CoreBot project can be removed, right?

Reply:
I can pull this, though there is a ticket that is being worked on to pull this from all the samples that use it.